tests/unit/test_utils.py

```diff
@@ -35,6 +35,10 @@ class MyUser(AbstractBaseUser):
     identity = models.CharField(max_length=50)
     USERNAME_FIELD = 'identity'
 
+    class Meta:
+        abstract = True
```

Comment:
Why is this important? I don't remember.

Reply:
If it's not important, feel free to drop it! 👍🏼️

Reply:
I don't know. I'm asking because I don't know.

The tests seem to work and seem to run for real, hence that seems okay. Still, if we know what every line of code does that would be preferable. Wouldn't it?

Reply:
> We don't want to install SQL for it, \[...\]

Source: [Django issue #419](https://code.djangoproject.com/ticket/419) (14 years ago)
Docs: [Models](https://docs.djangoproject.com/en/stable/topics/db/models/#abstract-base-classes)

I don't fully understand if that is relevant for the tests having the appropriate data to run or not. I worry only about that.